services/master/controllers/deployCheckers.go

```diff
@@ -0,0 +1,122 @@
+package controllers
+
+import (
+	"fmt"
+	"regexp"
+
+	"github.com/gofiber/fiber/v2"
+	"github.com/mholt/archiver/v3"
+	checker "github.com/sdslabs/katana/services/challengecheckerservice"
+)
+
+func DeployCheckers(c *fiber.Ctx) error {
```

Comment:
should it be called DeployCheckers? or DeployChecker

Reply:
Multiple checkers can be deployed at once, so I kept it DeployCheckers

Reply:
I get it but in totally there would be one checker for a challenge right?